From: Andrew Morton <akpm@linux-foundation.org>
Date: Tue, 11 Aug 2026 13:06:31 -0700
Subject: [PATCH] 
 tools-testing-selftests-mm-test-anonymous-page-offset-merge-behaviour-fix

tweak comment, per David

Cc: David Hildenbrand (Arm) <david@kernel.org>
Cc: "Lorenzo Stoakes (ARM)" <ljs@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 tools/testing/selftests/mm/merge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/mm/merge.c b/tools/testing/selftests/mm/merge.c
index cb7cdb6b7ada5..ee369ed4a9bf3 100644
--- a/tools/testing/selftests/mm/merge.c
+++ b/tools/testing/selftests/mm/merge.c
@@ -1324,7 +1324,7 @@ TEST_F(merge, anon_and_page_offset_mismatch_memfd)
 	ASSERT_NE(ptr, MAP_FAILED);
 
 	/*
-	 * Map another separately, and fault in, at page offset 5:
+	 * Map another separately and trigger a CoW fault, at page offset 5:
 	 *
 	 * |-----------|           |---------|
 	 * | unfaulted |           | faulted |
-- 
2.47.3

